### PR TITLE
Don't inline convert() calls to reduce binary size

### PR DESCRIPTION
**Pull request by kkaefer**

### CODE REVIEW

File: include/mbgl/util/util.hpp [hunk 1]
```diff
@@ -19,3 +19,10 @@
 #else
 #define MBGL_CONSTEXPR inline
 #endif
+
+// Control inlining
+#ifdef _MSC_VER
+#define MBGL_INLINE __declspec(noinline)
```
Comment (pozdnyakov) [inline include/mbgl/util/util.hpp]: MBGL_NOINLINE ?

Comment (kkaefer) [reply include/mbgl/util/util.hpp]: 🤦‍♂️

**Comment by ChrisLoer** [comment]
The approach looks good to me pending name change. I imagine the potential performance impact of not inlining could show up most in style parsing? Did you have any way of benchmarking before and after the changes? I tried running `mbgl-benchmark` before and after looking at all the old "parse" benchmarks (even though they're kind of a legacy pathway, they still exercise JSON parsing). I didn't see any big difference, but I also see large differences between runs of those benchmarks so I don't have a ton of confidence in them.

**Comment by ChrisLoer** [comment]
Wait... the latest binary size checks don't look like a win (except for Android AAR) -- what's going on there?

**Pull request closed without merging**